src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -81,6 +81,7 @@
 
     private static final String PREBID_EXT = "prebid";
     private static final String CONTEXT_EXT = "context";
+    private static final String SKAN_EXT = "skan";
```

Comment:
there was a typo in the original issue. this should be `skadn` instead of `skan`.